packages/web-runtime/tests/unit/pages/account.spec.js

```diff
@@ -0,0 +1,171 @@
+import account from '../../../src/pages/account.vue'
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import { createStore } from 'vuex-extensions'
+import GetTextPlugin from 'vue-gettext'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+
+localVue.directive('translate', {
+  inserted: (el) => {}
+})
+
+const $route = {
+  meta: {
+    title: 'Some Title'
+  }
+}
+
+const selectors = {
+  pageTitle: '.oc-page-title',
+  loaderStub: 'oc-loader-stub',
+  accountPageTitle: '#account-page-title',
+  editUrlButton: '[data-testid="account-page-edit-url-btn"]',
+  editRouteButton: '[data-testid="account-page-edit-route-btn"]',
+  accountPageInfo: '.account-page-info',
+  groupInformation: '.account-page-info-groups'
+}
+
+describe('account', () => {
+  describe('when the wrapper is still loading', () => {
+    let wrapper
+    beforeEach(() => {
+      wrapper = getWrapper()
+      wrapper.setData({ loading: true })
+    })
+    it('should render the loading state', () => {
+      const pageTitle = wrapper.find(selectors.pageTitle)
+      const ocLoader = wrapper.find(selectors.loaderStub)
+      expect(pageTitle.text()).toBe($route.meta.title)
+      expect(ocLoader.exists()).toBeTruthy()
+    })
+    it('should not render the account page content', () => {
+      const accountPageTitle = wrapper.find(selectors.accountPageTitle)
+      const accountPageInfo = wrapper.find('.account-page-info')
+      expect(accountPageTitle.exists()).toBeFalsy()
+      expect(accountPageInfo.exists()).toBeFalsy()
+    })
+  })
+  describe('when the wrapper is not loading anymore', () => {
+    it('should not render the loading state', async () => {
+      const store = getStore()
+      const wrapper = getWrapper(store)
+      await wrapper.setData({ loading: false })
+      const pageTitle = wrapper.find(selectors.pageTitle)
+      const ocLoader = wrapper.find(selectors.loaderStub)
+      expect(pageTitle.text()).toBe('Account')
```

Comment:
I'll allow it for this PR, but in general I don't like strings from the code hardcoded in tests. The implementation of the page title is wrong though, so let's keep it like it is, I'll adjust it in my PR mentioned before (if the code were correct you could use the title from your route mock, which makes the test code more resilient). Again, just FYI, nothing to do from your side.